### PR TITLE
[2.4] AAP-18144: Peer review edits (#845)

### PR DESCRIPTION
**Pull request by lmalivert**
Jira link: [AAP-18144](https://issues.redhat.com/browse/AAP-18144)

[AAP Install Guide](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/platform-system-requirements#doc-wrapper)

Incorporated editorial suggestions from [Chapter 2 System Requirements](https://docs.google.com/document/d/1tRNUZxvBDFxPbiPDVGWEywvwV2Eta_5mhk-CvbQIObc/edit#heading=h.hwwjfp2bkfct)

Files modified:

assemblies/platform/assembly-system-requirements.adoc
modules/platform/proc-benchmark-postgresql.adoc
modules/platform/proc-enable-hstore-extension.adoc
modules/platform/ref-automation-hub-requirements.adoc
modules/platform/ref-controller-system-requirements.adoc
modules/platform/ref-ha-hub-reqs.adoc
modules/platform/ref-postgresql-requirements.adoc
modules/platform/ref-system-requirements.adoc

### CODE REVIEW

File: downstream/modules/platform/proc-benchmark-postgresql.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ The examples shown in the procedure require at least 60GB disk space in the `/tm
 ** `size=10G` sets the file size generated by each job.
 --
 +
-To reduce the amount of test data, adjust the value of the `size` parameter.
+* You have adjusted the value of the `size` parameter. Adjusting this value reduces the amout of test data.
```
Comment (hherbly) [inline downstream/modules/platform/proc-benchmark-postgresql.adoc:26]: "Amount" is missing the "n".

File: downstream/modules/platform/ref-controller-system-requirements.adoc [hunk 2]
```diff
@@ -45,15 +40,31 @@ h| Requirement | Required
 | *Local disk* a|
 * 40GB minimum with at least 20GB available under /var/lib/awx
 * Storage volume must be rated for a minimum baseline of 1500 IOPS
-* Projects are stored on control and hybrid nodes, and for the duration of jobs, are also stored on execution nodes. If the cluster has many large projects, consider doubling the GB in /var/lib/awx/projects, to avoid disk space errors
+* Projects are stored on control and hybrid nodes, and for the duration of jobs, are also stored on execution nodes. If the cluster has many large projects, consider doubling the GB in /var/lib/awx/projects, to avoid disk space errors.
+|===
+
+*Hop nodes*
+
+Serve to route traffic from one part of the automation mesh to another (for example, could be a bastion host into another network). RAM could affect throughput, CPU activity is low. Netowrk bandwidth and latency are generally a more important factor than either RAM or CPU.
```
Comment (hherbly) [inline downstream/modules/platform/ref-controller-system-requirements.adoc:48]: Typo in "network" here.

File: downstream/modules/platform/ref-controller-system-requirements.adoc [hunk 3]
```diff
@@ -45,15 +40,31 @@ h| Requirement | Required
...
+* A larger number of hosts can of course be addressed, but if the fork number is less than the total host count, more passes across the hosts are required. You can avoid these RAM limitations by using any of the following approaches:
+** Use rolling updates.
+** Use the provisioning callback system built into {ControllerName}, whereeach system requesting configuration enters a queue and is processed as quickly as possible.
+** In cases where {ControllerName} is producting or deploying images such as AMIs.
 
 [role="_additional-resources"]
 .Additional resources
 
-* link:https://docs.ansible.com/automation-controller/latest/html/userguide/import_license.html?extIdCarryOver=true&sc_cid=7013a00000388B5AAI[Import a subscription].
+* For more informatino about obtaining an {ControllerName} subscription, see link:https://docs.ansible.com/automation-controller/latest/html/userguide/import_license.html?extIdCarryOver=true&sc_cid=7013a00000388B5AAI[Import a subscription].
```
Comment (hherbly) [inline downstream/modules/platform/ref-controller-system-requirements.adoc:69]: Typo in "information"

File: downstream/modules/platform/ref-system-requirements.adoc [hunk 4]
```diff
@@ -38,13 +38,13 @@ See link:https://docs.ansible.com/ansible/latest/getting_started/index.html[Gett
 
 .Additional notes for {PlatformName} requirements
 
-* Although {PlatformName} depends on Ansible Playbooks and requires the installation of the latest stable version of Ansible before installing {ControllerName}, manual installations of Ansible are no longer required.
+* {PlatformName} depends on Ansible Playbooks and requires the installation of the latest stable version of ansible-core. You can download ansible-core manually or download it automatically as part of your installation of {PlatformName}.
 
-* For new installations, {ControllerName} installs the latest release package of Ansible 2.14.
+* For new installations, {ControllerName} installs the latest release package of ansbile-core.
```
Comment (hherbly) [inline downstream/modules/platform/ref-system-requirements.adoc:43]: Typo in "ansible" here.

**Review by hherbly — APPROVED** [review]

File: downstream/modules/platform/ref-controller-system-requirements.adoc [hunk 5]
```diff
@@ -45,15 +40,31 @@ h| Requirement | Required
...
-* A larger number of hosts can of course be addressed, though if the fork number is less than the total host count, more passes across the hosts are required.
-These RAM limitations are avoided when using rolling updates or when using the provisioning callback system built into {ControllerName}, where each system requesting configuration enters a queue and is processed as quickly as possible; or in cases where {ControllerName} is producing or deploying images such as AMIs.
-All of these are great approaches to managing larger environments. For further questions, please contact Ansible support through the link:https://access.redhat.com/[Red Hat Customer portal].
+* A larger number of hosts can of course be addressed, but if the fork number is less than the total host count, more passes across the hosts are required. You can avoid these RAM limitations by using any of the following approaches:
+** Use rolling updates.
+** Use the provisioning callback system built into {ControllerName}, whereeach system requesting configuration enters a queue and is processed as quickly as possible.
+** In cases where {ControllerName} is producting or deploying images such as AMIs.
```
[the remark this answers is not in the document]
Comment (lmalivert) [reply downstream/modules/platform/ref-controller-system-requirements.adoc:64]: @hherbly Typo. It's supposed to be "producing." Good catch!

**Pull request merged**